charts/eg/crds/generated/config.gateway.envoyproxy.io_envoyproxies.yaml

```diff
@@ -1,239 +0,0 @@
----
```

Comment:
can you please remind me why this  isn't needed anymore

Reply:
I think it's moved to charts/gateway-helm and will not be update by `make generate manifests`

Reply:
good catch, so `charts/eg` is not longer needed ? 
cc @Xunzhuo